docker-compose.yml

```diff
@@ -144,8 +146,19 @@ services:
             - SCREEN_DEPTH=24
         depends_on:
             - seleniumhub
+
+    docker:
+        build: docker
         networks:
             - omero-network
+        volumes:
+            - /var/run/docker.sock:/var/run/docker.sock
+            - ${REPO_CURATED}:/repo/curated:ro
+            - ${REPO_CONFIG}:/repo/config:ro
```

Comment:
Here `REPO_CURATED` is used to mount the volumes but below it is read as an environment variable (which currently returns `null`). Is the expectation that the job use `/repo/{config,curated}`

Reply:
I see what you mean. There's not reason to be _that_ flexible. Agreed, internally let's just _define_ the location to be `/repo/{config,curated}` and then docker will handle the mapping.